features/aanhef.feature

```diff
@@ -2,216 +2,762 @@
+      | BS                      | P                               | P           | Hoogheid                              |
+      | BS                      | P                               | V           | Hoogheid                              |
+      | BS                      | P                               | N           | Hoogheid                              |
+      | BS                      | R                               | E           | Hoogwelgeboren vrouwe                 |
+      | BS                      | R                               | P           | Geachte mevrouw De Boer               |
+      | BS                      | R                               | V           | Hoogwelgeboren vrouwe                 |
+      | BS                      | R                               | P           | Hoogwelgeboren vrouwe                 |
+
+  Abstract Scenario: man heeft adellijke titel <adellijkeTitelPredikaat> en zijn partner heeft adellijke titel "<adellijkeTitelPredikaat partner>" bij aanduidingNaamgebruik "<naamgebruik>"
+    Gegeven het systeem heeft een persoon met de volgende gegevens
+    | naam                | waarde     |
+    | burgerservicenummer | 999992934  |
+    | geslachtsaanduiding | M          |
+    En de persoon heeft de volgende naam gegevens
+    | naam                                 | waarde        |
+    | voornamen (02.10)                    | Jo            |
+    | adellijke titel of predikaat (02.20) |               |
+    | voorvoegsel (02.30)                  | van den       |
+    | geslachtsnaam (02.40)                | Aedel         |
+    | aanduidingNaamgebruik (61.10)        | <naamgebruik> |
+    En de persoon heeft een partner met de volgende gegevens
+    | naam                        | waarde    |
+    | geslachtsaanduiding (04.10) | V         |
+    En de partner heeft de volgende naam gegevens
+    | naam                                 | waarde                            |
+    | adellijke titel of predikaat (02.20) | <adellijkeTitelPredikaat partner> |
+    | voorvoegsel (02.30)                  | de                                |
+    | geslachtsnaam (02.40)                | Boer                             |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 999992934                       |
+    | fields              | naam.aanhef                     |
+    Dan bevat de persoon met burgerservicenummer '999992934' de volgende 'naam' gegevens
+    | naam   | waarde   |
+    | aanhef | <aanhef> |
+
+    Voorbeelden:
+      | adellijkeTitelPredikaat | adellijkeTitelPredikaat partner | naamgebruik | aanhef               |
+      | G                       | BS                              | E           | Hooggeboren heer     |
+      | G                       | BS                              | P           | Geachte heer De Boer |
```

Comment:
Deze is volgens mij nergens in de Rule beschreven. Volgens mij zou er een rule moeten worden beschreven waarin uitgelegd wordt dat de aanhef voor een man niet wordt gebaseerd op de adellijke titel  van zijn partner maar dat hij dan  de aanhef krijgt alsof er geen adellijke titel of predikaat is.

Reply:
dat de man de adellijke titel van zijn partner niet mag gebruiken staat denk ik wel al in een regel:
> Rule: De aanhef voor een persoon wordt bepaald door het adellijkeTitelPredikaat van de (ex)partner onder de volgende condities:
>   - de (ex)partner heeft een adellijke titel
>   - de geslachtsaanduiding van de persoon is "V"
>   - de geslachtsaanduiding van de partner is "M"

De reden dat in dit geval niet de adellijke titel van de persoon zelf wordt gebruikt is dat naamgebruik=P (partner). Dit staat op regel 200:

>   - Wanneer de persoon de geslachtsnaam van de echtgenoot/partner gebruikt zonder de eigen geslachtsnaam: aanduidingNaamgebruik is "P"

De andere situaties (hieronder) tonen namelijk dat bij aanduidingNaamgebruik ongelijk aan "P" wel de aanhef gebaseerd is op de adellijke titel van de persoon zelf